Tests/ClientRuntimeTests/NetworkingTests/HttpRequestTests.swift

```diff
@@ -51,6 +57,31 @@ class HttpRequestTests: NetworkingTestUtils {
         }
     }
 
+    func testSdkHttpRequestToURLRequest() async throws {
+        let headers = Headers(["Testname-1": "testvalue-1", "Testname-2": "testvalue-2"])
+        let endpoint = Endpoint(host: "host.com", path: "/", headers: headers)
+
+        let httpBody = HttpBody.data(expectedMockRequestData)
+        let mockHttpRequest = SdkHttpRequest(method: .get, endpoint: endpoint, body: httpBody)
+        let urlRequest = try await mockHttpRequest.toURLRequest()
+
+        XCTAssertNotNil(urlRequest)
+        guard let headersFromRequest = urlRequest.allHTTPHeaderFields else {
+            XCTFail("Headers in SdkHttpRequest were not successfully converted to headers in URLRequest.")
+            // Compiler doesn't recognize XCTFail as return / exception thrown
+            return
+        }
+        XCTAssertNotNil(headersFromRequest)
```

Comment:
Is this check necessary?  `headersFromRequest` is necessarily non-nil here because of the `guard` statement above

Reply:
That's true, changed as suggested.